rune/libcontainer/nsenter/nsexec.c

```diff
@@ -698,10 +701,15 @@ void nsexec(void)
 			int len;
 			pid_t child, first_child = -1;
 			bool ready = false;
+			int ret;
 
 			/* For debugging. */
 			prctl(PR_SET_NAME, (unsigned long)"runc:[0:PARENT]", 0, 0, 0);
 
+			ret = pal_preload();
```

Comment:
Rename to load_enclave_runtime().

Reply:
In addition, you need to call `prctl(PR_SET_NAME, (unsigned long)"init-runlet", 0, 0, 0)` by grandson process on the exit path of nsexec.c.